src/client/components/App.tsx

```diff
@@ -72,24 +77,25 @@ const App: FunctionComponent = (): ReactElement => {
   useEffect((): void => {
     if (dispatchMessageRef.current) {
       const { current: dispatchMessage } = dispatchMessageRef;
+      setDataFetching(true);
       getCurrentUser()
         .then((user: User): void => {
-          if (user) {
-            setUser(user);
-            getMetadata()
-              .then((metadata: MetadataResponse): void => {
-                setMetadata(metadata);
-              })
-              .catch((): void => {
-                dispatchMessage({
-                  message: new AppMessage(
-                    'Unable to get metadata from server. If the problem persists, contact SEAS Computing',
-                    MESSAGE_TYPE.ERROR
-                  ),
-                  type: MESSAGE_ACTION.PUSH,
-                });
+          setUser(user);
```

Comment:
I think this slipped through from when I had started implementing the public view for the app, since we will have cases where we won't necessarily get a `User` back from the API method. That's not really applicable here, but if we do get a `null` value back from the API method it just falls through to the `AppRouter` and shows the `401 Unauthorized` error page.

I'm inclined to leave it here, since we'll be implementing the public view next and it's not breaking anything, but I'd also be OK putting it back since it probably shouldn't have been changed as part of this PR. What do you think?

Reply:
I see - I think that since it's not really affecting anything right now and the implementation of the public view is next, it should be ok as it is.